### PR TITLE
Fix model dependency

### PR DESCRIPTION
**Pull request by jenbutongit**
# Description

Model was not being consumed by the runner correctly and would throw this error when running `yarn runner start|dev|etc`
```
TypeError: Cannot read property 'validate' of undefined
    at new FormModel ($repo_dir/digital-form-builder/runner/dist/server/plugins/engine/models/FormModel.js:50:34)
```

Not sure _why_ exactly, but it did not like ./schema being exported twice. possibly a naming issue?? 

- Put schema/schema.ts into schema/index.ts, since there are no other related exports

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce
the testing if necessary.

Before PR's can be merged they will need to be tested by QA and approved where
applicable. To flag the change to QA assign **@XGovFormBuilder/qa** as one of the reviewers.

- [x] after `yarn model build && yarn && yarn runner start` the error is no longer thrown 

# Checklist:

- [x] My changes do not introduce any new linting errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and versioning
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have rebased onto master and there are no code conflicts
- [x] I have checked deployments are working in all environments
- [x] I have updated the architecture diagrams as per Contribute.md

### CODE REVIEW

File: model/src/schema/index.ts [hunk 1]
```diff
@@ -1 +1,261 @@
-export { Schema } from "./schema";
```
Comment (bertrandmc) [inline model/src/schema/index.ts:1]: Why are you renaming schema.ts to index.ts?

**Review by bertrandmc — CHANGES_REQUESTED** [review]

File: model/src/schema/index.ts (hunk 1 above)
Comment (jenbutongit) [reply model/src/schema/index.ts:1]: see description

Comment (bertrandmc) [reply model/src/schema/index.ts:1]: Sorry Jen, I can't reproduce the bug you mention and it is weird that index is being used for encapsulation everywhere but fails only in schema, should we investigate better? Is this a production bug?

Comment (bertrandmc) [reply model/src/schema/index.ts:1]: @jenbutongit I've just tried it again and really can't reproduce the bug, both locally and inside doker.

Comment (jonnywyatt) [reply model/src/schema/index.ts:1]: @bertrandmc for me, the `rm -rf model/dist` before building fixed it. The problem seemed to be that `yarn model build` didn't overwrite the existing files in `model/dist`

**Comment by jenbutongit** [comment]
resolved, thanks all!

**Pull request closed without merging**